packages/core/src/plugins/stream/SoundcloudPlugin.ts

```diff
@@ -38,13 +41,7 @@ class SoundcloudPlugin extends StreamProviderPlugin {
   }
 
   async getStreamForId(id: string): Promise<undefined | StreamData> {
-    try { 
-      const result = await(await Soundcloud.getTrackById(id)).json();
-      return result.id && this.resultToStream(result);
-    } catch (err) {
-      logger.error(`Error while looking up streams id: ${id} on Soundcloud`);
-      logger.error(err);
-    }
+    return null;
```

Comment:
Sound cloud API no longer supports track ids - so returning null here for now. Any thoughts on this?